### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/config/frontmatter/layout.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#554] [Announcements]
# https://theme-hope.vuejs.press/zh/config/frontmatter/layout.html

你可以在页面的 frontmatter 配置以下选项控制页面布局。 pageInfo 类型: PageInfo[] | false; 默认值: 主题选项中的值; PageInfo 可选的值和对应内容如下: 条目 对应内容 页面 frontmatter 值 --------------- ------------ --------------------...

https://theme-hope.vuejs.press/zh/config/frontmatter/layout.html

**Comment by dragon6long** [comment]
lastUpdate为什么没有用啊请问，我打包到服务器最底部还是会显示
